assets/src/block-editor/helpers/index.js

```diff
@@ -355,14 +359,37 @@ export const setImageBlockLayoutAttributes = ( props, layout ) => {
  * @return {ReactElement} Inspector Controls.
  */
 export const setUpInspectorControls = ( props ) => {
```

Comment:
How about creating a new `setUpVideoInspectorControls` function that would be selected in the `filterBlocksEdit` function whenever we're dealing with the `core/video` block, i.e.:

```diff
		if ( 'core/gallery' === name ) {
			inspectorControls = setUpGalleryInspectorControls( props );
		} else if ( 'core/image' === name ) {
			inspectorControls = setUpImageInspectorControls( props );
-		} else if ( MEDIA_BLOCKS.includes( name ) || 0 === name.indexOf( 'core-embed/' ) ) {
+		} else if ( 'core/video' === name ) {
+			inspectorControls = setUpVideoInspectorControls( props );
+		} else if ( 0 === name.indexOf( 'core-embed/' ) ) {
			inspectorControls = setUpInspectorControls( props );
		}
```

Note that since `MEDIA_BLOCKS` consists of only the `core/image` and `core/video` elements, there would be no need to include the check in the last `else if` statement.